src/extras/BitmapText.js

```diff
@@ -484,16 +486,16 @@ export default class BitmapText extends core.Container
             const charCode = parseInt(letter.getAttribute('id'), 10);
 
             const textureRect = new core.Rectangle(
-                parseInt(letter.getAttribute('x'), 10) + texture.frame.x,
-                parseInt(letter.getAttribute('y'), 10) + texture.frame.y,
-                parseInt(letter.getAttribute('width'), 10),
-                parseInt(letter.getAttribute('height'), 10)
+                parseInt(letter.getAttribute('x'), 10) / res + texture.frame.x / res,
```

Comment:
Could we get some brackets around the last division on this line and the next to make the bodmas ordering explicit. I thought we had an eslint rule that enforced this (http://eslint.org/docs/rules/no-mixed-operators)